### PR TITLE
Remove the discovery functionality in the Relay node

### PR DESCRIPTION
**Pull request by vishnoianil**
Given that we have the stun based discovery in the agent itself, relay based discovery function is now
 redundant.

### CODE REVIEW

**Comment by vishnoianil** [comment]
> lgtm so far, pending CI and whatever was on your todo list to remove it from draft

wanted to run local e2e and few more tests before opening the PR. Also updated the related bits in the documents.

File: docs/user-guide/discovery-and-relay.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
 # Deploying the Nexodus Discovery and Relay Nodes
```
Comment (nerdalert) [inline docs/user-guide/discovery-and-relay.md:1]: Pull `Discovery` from the header if you dont mind.

**Review by nerdalert — APPROVED** [review]
LGTM, will let @russellb merge when he is g2g. Ty.

Passed ec2 dev tests as well: https://github.com/nexodus-io/nexodus/actions/runs/4748830461/jobs/8435618856

**Review by russellb — APPROVED** [review]
Brent had one minor doc request. Feel free to merge after that.

File: docs/user-guide/discovery-and-relay.md (hunk 1 above)
Comment (vishnoianil) [reply docs/user-guide/discovery-and-relay.md]: Thanks for catching that. Fixed.

**Pull request merged**